echo-rest/src/main/java/com/netflix/spinnaker/echo/events/RestEventListener.java

```diff
@@ -39,7 +39,7 @@
 @ConditionalOnProperty("rest.enabled")
 @Getter
 @Setter
-class RestEventListener implements EventListener {
+public class RestEventListener implements EventListener {
```

Comment:
What's the motivation for this?

Reply:
Forgot to add a package to the test class. Fixed https://github.com/spinnaker/echo/commit/4568a320306ba2b5f897f9833d64c321437f02e3